websocket/_app.py

```diff
@@ -182,6 +185,8 @@ def __init__(self, url, header=None,
         self.last_ping_tm = 0
         self.last_pong_tm = 0
         self.subprotocols = subprotocols
+        
+        self.prepared_socket = socket
```

Comment:
I didn't use `self.socket` because we're already had `self.sock`, so I went with `self.prepared_socket`.